app/assets/stylesheets/card.scss

```diff
@@ -11,15 +12,24 @@
       color: $govuk-text-colour;
     }
 
+    .hf-card-details h2 {
```

Comment:
There is duplicate styling here for the h2 and h3. This can be condensed so it is:
`.hf-card-details h2, .hf-card-details h3 { ... }`
or
`.hf-card-details h2, h3 { ... }`

Reply:
the light blue link bit?